content/docs/react-without-jsx.md

```diff
@@ -22,8 +22,7 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
-
-can be compiled to this code that does not use JSX:
+se puede compilar a este que no usa JSX:
```

Comment:
Don't change the spacing in the line numbers in the original article. It makes our lives easier for reviewing.
```suggestion

se puede compilar a este que no usa JSX:
```